src/dataExplorer/components/DataExplorer.tsx

```diff
@@ -16,16 +16,55 @@ import {queryBuilderFetcher} from 'src/timeMachine/apis/QueryBuilderFetcher'
 import {readQueryParams} from 'src/shared/utils/queryParams'
 import ErrorBoundary from 'src/shared/components/ErrorBoundary'
 
+import {getActiveTimeMachine} from 'src/timeMachine/selectors'
+
+import {getStore} from 'src/store/configureStore'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+;(() => {
+  if (isFlagEnabled('persistRefresh')) {
```

Comment:
no action required, but if you find yourself in a place where everything is in a condition, it sometimes makes cleaner code to invert that condition and return early. ie:
```
if (!isFlagEnabled('persistRefresh')) {
  return
}

// do the rest of your magic here
```